### PR TITLE
:memo: Add onboarding information

### LINKED ISSUE #258: Add onboarding information
Stuff we could borrow from [Notts](https://cdu-data-science-team.github.io/the-cdu-way/docs/introduction/)
- Data Science trophy :trophy:
- Flexible working / working times
- Communication slack / teams

And the document Chris wrote for when I joined

### LINKED ISSUE #259: Add onboarding information
Stuff we could borrow from [Notts](https://cdu-data-science-team.github.io/the-cdu-way/docs/introduction/)
- Data Science trophy :trophy:
- Flexible working / working times
- Communication slack / teams

And the document Chris wrote for when I joined

### PR DESCRIPTION
**Pull request by StatsRhian**
Fixes #259

Have added some basic things about the way we work as a team.
All suggestions welcome.

Screen grab of local rendered version
rendered

### CODE REVIEW

**Review by yiwen-h — APPROVED** [review]

File: style/team.qmd [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Your line manager should arrange (1 : 1) meetings with you once a fortnight.
+
+## Communication
+
+We use MS Teams to communicate about project related work. 
+We have a private slack channel on NHS-R slack which we use for general team chat and more casually. (Slack has much better emojis 🤸🎮🦥)
+
+## Data Science Trophy 
+
+One of the ways we celebrate each other and the successes we have is by presenting each other with a “trophy” 🏆 in the slack channel. This is for good code, problem solving, helping someone anything that deserves a special mention.
```
Comment (ChrisBeeley) [inline style/team.qmd:26]:
```suggestion
One of the ways we celebrate each other and the successes we have is by presenting each other with a “trophy” 🏆 in the slack channel. This is for good code, problem solving, helping someone- anything that deserves a special mention.
```

Comment (ChrisBeeley) [inline style/team.qmd:26]:
```suggestion
One of the ways we celebrate each other and the successes we have is by presenting each other with a “trophy” 🏆 in the slack channel. This is for good code, problem solving, helping someone- anything that deserves a special mention.
```

File: style/team.qmd [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Team work
+
+## Weekly stand up
+
+Every Monday from 10:00 - 11:00 we have a team meeting. It starts with everyone answering the following questions:
```
Comment (ChrisBeeley) [inline style/team.qmd:5]:
```suggestion
Every Monday from 10:00 - 10:50 we have a team meeting. It starts with everyone answering the following questions:
```

File: style/team.qmd [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Tasks outside this project can be managed how you like. If you want to use GitHub, there is a [Data Science project planner](https://github.com/orgs/The-Strategy-Unit/projects/12).
+
+## Coffee and Code
+
+We organise a fortnightly coffee and code for analysts in the Strategy Unit. This is planned in a [Coffee and Coding ☕🧑‍💻 GitHub project](https://github.com/orgs/The-Strategy-Unit/projects/14). Materials from previous sessions are shared here as [blogs](../blogs/) or [presentations](../presentations/).
+
+## Sickness
+
+It is an acceptable to communicate things like sickness or last-minute absences via the team slack channel.
```
Comment (ChrisBeeley) [inline style/team.qmd:48]: Not sure what is meant by this. Can you rephrase?

File: style/team.qmd [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+We use MS Teams to communicate about project related work. 
+We have a private slack channel on NHS-R slack which we use for general team chat and more casually. (Slack has much better emojis 🤸🎮🦥)
+
+## Data Science Trophy 
+
+One of the ways we celebrate each other and the successes we have is by presenting each other with a “trophy” 🏆 in the slack channel. This is for good code, problem solving, helping someone anything that deserves a special mention.
+
+## Working Times 
+
+Core hours vary considerably between team members and there is a lot of flexibility. Team members may send messages out of hours, but that is purely because it works for them, and will not expect anything immediate in response. Flexibility is not only allowed but recommended if it benefits you.
```
Comment (ChrisBeeley) [inline style/team.qmd:30]:
```suggestion
Core hours vary considerably between team members and there is a lot of flexibility. Team members may send messages out of hours, but that is purely because it works for them, and will not expect anything immediate in response. Flexibility is not only allowed but recommended if it benefits you. Please agree core hours with your line manager and where possible communicate any changes in your core hours to all team members via Slack (e.g. "I'm out all morning because my car is broken- working later on").
```

**Review by ChrisBeeley — CHANGES_REQUESTED** [review]
Looks great, thanks. A couple of changes and comments. I hope the core hours bit doesn't sound too strong, let me know what you think (I don't mean it like you _must_ do this, more like just can be useful to let people know what you're up to)

File: style/team.qmd (hunk 3 above)
Comment (StatsRhian) [reply style/team.qmd:48]: This was from your Notts one. Is there anything you wanted to add on sickness? Or I can just remove?

I think the point you were trying to make on the Notts one was it's okay to ping people informally with last minute illness (and then follow up with the formal stuff after if needed)

File: style/team.qmd (hunk 4 above)
Comment (StatsRhian) [reply style/team.qmd:30]: I edited to suggest using out-of-office in calendar as well for communicating changes to core hours.

File: style/team.qmd [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+Core hours vary considerably between team members and there is a lot of flexibility. Team members may send messages out of hours, but that is purely because it works for them, and will not expect anything immediate in response. Flexibility is not only allowed but recommended if it benefits you. 
+
+## Working Location
+
+The team is primarily remote. The Strategy Unit has two all company gathering a year, in Birmingham. Currently the Data Science team don't have any in-person meetings, although we might start doing them occasionally when it's beneficial. Some of the London-based team coordinate when they work in the office.
+
+## Task management
+
+Most of the teams time is working on a large project which we manage in an agile-like manner. We use GitHub to capture issues, and a project board to manage the sprint.
```
Comment (matt-dray) [inline style/team.qmd:38]: lolol, nanocommit.

```suggestion
Most of the team's time is working on a large project which we manage in an agile-like manner. We use GitHub to capture issues, and a project board to manage the sprint. 
```

Comment (StatsRhian) [reply style/team.qmd]: 🐁

**Review by tomjemmett — APPROVED** [review]

File: style/team.qmd (hunk 3 above)
Comment (ChrisBeeley) [reply style/team.qmd:48]: Oh, lol, it's not grammatical, there must be a typo or something. I guess it's supposed to say "It is acceptable..."

Let's maybe just remove it because it's maybe just confusing when you have to say "Don't forget to follow up later..."

File: style/team.qmd [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+Tasks outside this project can be managed how you like. If you want to use GitHub, there is a [Data Science project planner](https://github.com/orgs/The-Strategy-Unit/projects/12).
+
+## Coffee and Code
+
+We organise a fortnightly coffee and code for analysts in the Strategy Unit. This is planned in a [Coffee and Coding ☕🧑‍💻 GitHub project](https://github.com/orgs/The-Strategy-Unit/projects/14). Materials from previous sessions are shared here as [blogs](../blogs/) or [presentations](../presentations/).
+
+## Sickness
+
+It is an acceptable to communicate things like sickness or last-minute absences via the team slack channel.
```
Comment (StatsRhian) [inline style/team.qmd:48]:
```suggestion
```

**Review by ChrisBeeley — APPROVED** [review]
